### ISSUE TITLE
custom marker class / styling issue (1.17.1 regression)

### ISSUE THREAD

**Issue by flc** [allenhwkim/angularjs-google-maps#558]
http://plnkr.co/edit/Jcjl1LGTJlcGo8t3q3SN?p=preview

change the ng map version from 1.17.1 to 1.16.7
```<script src="https://rawgit.com/allenhwkim/angularjs-google-maps/v1.16.7/build/scripts/ng-map.min.js"></script>```

as you can see the styling of the custom markers changes.

I think it's related to this commit: https://github.com/allenhwkim/angularjs-google-maps/commit/101e9b1084e519d7bd79aa31dfc1df66872c4f9a

**Comment by flc** [comment]
if you inspect the html of the first custom marker

v1.17.1
check the class(es) of the outer div
```
<div class="custom-marker {{ marker.label }}" style="display: inline-block; visibility: visible; position: absolute; left: 223px; top: 137px;">
        <div class="blue">
          <div class="marker-label ng-binding">
            blue
          </div>
          <div class="marker-content ng-binding">
            marker1
          </div>
        </div>
      </div>
```

v1.16.7
check the class(es) of the outer div
```
<div class="custom-marker blue" style="display: inline-block; visibility: visible; position: absolute; left: 223px; top: 137px;">
        <div class="blue">
          <div class="marker-label ng-binding">
            blue
          </div>
          <div class="marker-content ng-binding">
            marker1
          </div>
        </div>
      </div>
```

**Comment by allenhwkim** [comment]
Thanks for reporting it. I really appreciate it. I would be careful when I accept PR.

**Comment by allenhwkim** [comment]
I have fixed this I believe with 1.17.2
